jekyll/_cci2/configuration-reference.md

```diff
@@ -341,12 +341,19 @@ jobs:
 ##### Available `machine` images
 CircleCI supports multiple machine images that can be specified in the `image` field:
 
+* `ubuntu-2004:202101-01` - Ubuntu 20.04, Docker v20.10.2, Docker Compose v1.28.2,
 * `ubuntu-2004:202010-01` - Ubuntu 20.04, Docker v19.03.13, Docker Compose v1.27.4, `ubuntu-2004:202008-01` is an alias
+* `ubuntu-1604:202101-01` - Ubuntu 16.04, Docker v20.10.2, Docker Compose v1.28.2, 2nd to last release
 * `ubuntu-1604:202010-01` - Ubuntu 16.04, Docker v19.03.13, Docker Compose v1.27.4
 * `ubuntu-1604:202007-01` - Ubuntu 16.04, Docker v19.03.12, Docker Compose v1.26.1
 * `ubuntu-1604:202004-01` - Ubuntu 16.04, Docker v19.03.8, Docker Compose v1.25.5
 * `ubuntu-1604:201903-01` - Ubuntu 16.04, Docker v18.09.3, Docker Compose v1.23.1
 
+***Note:*** *Ubuntu 16.04 is being EOL'd by Canonical in April 2021.
+This also means that we will stop creating new releases with this version of Ubuntu.
+The last Ubuntu 16.04 machine image we will create will be the Q2 image in April.
+We suggest upgrading to the Ubuntu 20.04 image for continued releases past April.*
+
 The machine executor supports [Docker Layer Caching]({{ site.baseurl }}/2.0/docker-layer-caching) which is useful when you are building Docker images during your job or Workflow.
```

Comment:
Just some suggested text as EOL and Q2 _might_ not be understood. Take what you want!

> Ubuntu Linux 16.04 reaches the end of its LTS window at the end of April 2021 and will no longer be supported by Canonical. As a result, the final 16.04 CircleCI machine image release by us will take place in April to include the most recent security patches. We suggest upgrading to the Ubuntu 20.04 image for continued releases past April. 2021.

Reply:
I like it.